src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -74,12 +81,6 @@ public void setDrivePowers(double yScale, double angularScale) {
 
   public void setDrivePowers(double yScale, double angularScale, boolean squareInput) {
```

Comment:
If we're just passing it to DifferentialDrive, squareInput probably does not need to be a param (it can be a state instead).

Also.. still not so convinced on switching to DifferentialDrive

Reply:
I believe I did make it a state like `yScale` and `angularScale`; do you mean not taking it as an argument to that method at all and instead having some other way to set `squareInput`?

As for DifferentialDrive, yeah, perhaps we don't do that. I can undo the `DifferentialDrive` changes in favor of only changing `WPI_TalonSRX`.

Reply:
Yeah, and that 'other way to set squareInput' is TankSubsystem.squareInput = false